lib/membrane_mp4/muxer.ex

```diff
@@ -0,0 +1,149 @@
+defmodule Membrane.MP4.Muxer do
```

Comment:
I think we need to figure something out with the naming - now it is inconsistent with `CMAF.Muxer` and `Muxer` being on different "naming levels". The simplest thing to do I think would be `MP4.Muxer` and `MP4.CMAFMuxer`. It would be best to name the regular with some prefix as well, e.g. `MP4.IsomMuxer`. If you have an idea for the prefix, we could also go with `MP4.Muxer.CMAF` and `MP4.Muxer.Isom`

Reply:
I went with `Membrane.MP4.Muxer.{CMAF, ISOM}`, don't have any better idea for the prefix for a regular MP4 🤔